File: Baekjoon/C/10101.c
#include <stdio.h>
int main(){
	int a, b, c;
	scanf("%d", &a);
	scanf("%d", &b);
	scanf("%d", &c);
	
	if(a+b+c!=180){
		printf("Error");
	}else{
		if(a==b){
			if(a!=c){
				printf("Isosceles");
			}else{
				printf("Equilateral");
			}
		}else{
			if(a==c){
				printf("Isosceles");
			}else if(b==c){
				printf("Isosceles");
			}else{
				printf("Scalene");
			}
		}
	}
}

File: Baekjoon/Python/10101.py
angular = []

for _ in range(3):
    angular.append(int(input()))

sum_ = sum(angular)
set_ = set(angular)

if sum_ == 180:
    if len(set_) == 1:
        print('Equilateral')
    elif len(set_) == 2:
        print('Isosceles')
    else:
        print('Scalene')
else:
    print('Error')
